src/sherpa_ai/config/task_config.py

```diff
@@ -21,7 +24,14 @@ def from_input(cls, input_str: str) -> Tuple[str, "AgentConfig"]:
 
         for part in parts[1:]:
             part = part.strip()
-            configs.extend(part.split())
+            if part.startswith("--gsite"):
```

Comment:
This is the purpose of this part, it seems to be a bit redundant to the part starting at line 64